Sources/BraveWallet/Crypto/Scanner/AddressQRCodeScannerView.swift

```diff
@@ -238,3 +260,19 @@ extension AddressQRCodeScannerViewController: AVCaptureMetadataOutputObjectsDele
     isFinishScanning = false
   }
 }
+
+private extension String {
```

Comment:
@StephenHeaps if it is urgent you can do it later but can you add a test for this ?

Reply:
fixed [beb19ae](https://github.com/brave/brave-ios/pull/6088/commits/beb19ae39c2fc3782f09f73e926d0257980b9b9d)

Reply:
🔥